libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Security/SFAuthenticationManager.m

```diff
@@ -1400,6 +1403,11 @@ - (void)oauthCoordinator:(SFOAuthCoordinator *)coordinator displayConfirmationMe
     [[SFRootViewManager sharedManager] pushViewController:alert];
 }
 
+- (void)oauthCoordinatorDidCancelBrowserAuthentication:(SFOAuthCoordinator*)coordinator
+{
+    [self delegateDidCancelBrowserFlow];
```

Comment:
This gets called from [here](https://github.com/wmathurin/SalesforceMobileSDK-iOS/blob/805fbd1e344a0c630e879af09a1fae58fa253eb7/libs/SalesforceSDKCore/SalesforceSDKCore/Classes/OAuth/SFOAuthCoordinator.m#L1025). Is that what you are asking?